tests/components/recorder/test_init.py

```diff
@@ -192,8 +200,8 @@ def test_saving_state_include_domain_exclude_entity(hass_recorder):
     )
     states = _add_entities(hass, ["test.recorder", "test2.recorder", "test.ok"])
     assert len(states) == 1
-    assert hass.states.get("test.ok") == states[0]
-    assert hass.states.get("test.ok").state == "state2"
+    assert _state_empty_context(hass, "test.ok") == states[0]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5745133)